doc/releases/changelog-0.30.0.md

```diff
@@ -149,53 +288,115 @@
-  [(3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
+* The QNode keyword argument `mode` has been replaced by the boolean `grad_on_execution`.
+  [(#3969)](https://github.com/PennyLaneAI/pennylane/pull/3969)
+
+* The `"default.gaussian"` device and parameter-shift CV both support the new return system, but only for single measurements.
+  [(#3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
 
 * Keras and Torch NN modules are now compatible with the new return type system.
   [(#3913)](https://github.com/PennyLaneAI/pennylane/pull/3913)
   [(#3914)](https://github.com/PennyLaneAI/pennylane/pull/3914)
 
-* `DefaultQutrit` supports the new return system.
+* `DefaultQutrit` now supports the new return system.
   [(#3934)](https://github.com/PennyLaneAI/pennylane/pull/3934)
 
-  [(3946)](https://github.com/PennyLaneAI/pennylane/pull/3946)
-
 <h4>Other improvements</h4>
 
-* Added a `Shots` class to the `measurements` module to hold shot-related data.
+* A `Shots` class has been added to the `measurements` module to hold shot-related data.
   [(#3682)](https://github.com/PennyLaneAI/pennylane/pull/3682)
 
-* The `coefficients` function and the `visualize` submodule of the `qml.fourier` module
-  now allow assigning different degrees for different parameters of the input function.
-  [(#3005)](https://github.com/PennyLaneAI/pennylane/pull/3005)
-
-  The arguments `degree` and `filter_threshold` to `qml.fourier.coefficients` previously were
-  expected to be integers, and now can be a sequences of integers with one integer per function
-  parameter (i.e. `len(degree)==n_inputs`), resulting in a returned array with shape
-  `(2*degrees[0]+1,..., 2*degrees[-1]+1)`.
-  The functions in `qml.fourier.visualize` accordingly accept such arrays of coefficients.
-
-* `Operator` now has a `has_generator` attribute that returns whether or not the operator
-  has a generator defined. It is used in `qml.operation.has_gen`, improving its performance.
-  [(#3875)](https://github.com/PennyLaneAI/pennylane/pull/3875)
-
 * The custom JVP rules in PennyLane now also support non-scalar and mixed-shape tape parameters as
   well as multi-dimensional tape return types, like broadcasted `qml.probs`, for example.
   [(#3766)](https://github.com/PennyLaneAI/pennylane/pull/3766)
 
-* The `qchem.jordan_wigner` function is extended to support more fermionic operator orders.
+* The `qchem.jordan_wigner` function has been extended to support more fermionic operator orders.
   [(#3754)](https://github.com/PennyLaneAI/pennylane/pull/3754)
   [(#3751)](https://github.com/PennyLaneAI/pennylane/pull/3751)
 
-* `AdaptiveOptimizer` is updated to use non-default user-defined qnode arguments.
+* `AdaptiveOptimizer` has been updated to use non-default user-defined QNode arguments.
   [(#3765)](https://github.com/PennyLaneAI/pennylane/pull/3765)
 
 * Use `TensorLike` type in `Operator` dunder methods.
```

Comment:
🤔

Reply:
It's stuff like this: https://github.com/PennyLaneAI/pennylane/pull/3749/files#diff-ed5f9c0de17892f74e5119e2a032fe277eaa330ec4df0c249c7a75af7be6585eR1413

In the dunder methods (add, multiply, etc.) we sometimes need to know if one of the operators is tensor-like (i.e., a tensor product of sub-operators). This PR used the general `TensorLike` class to detect this.

Does that make sense? Not sure if we need to update in the changelog.